src/SimpleGraphs/generators/staticgraphs.jl

```diff
@@ -763,3 +763,27 @@ function lollipop_graph(n1::T, n2::T) where {T <: Integer}
     add_edge!(g, n1, n1+1)
     return g
 end
+
+
+"""
+    friendship_graph(n)
+Create a [friendship graph](https://en.wikipedia.org/wiki/Friendship_graph)
```

Comment:
```suggestion
    friendship_graph(n)
    
Create a [friendship graph](https://en.wikipedia.org/wiki/Friendship_graph)
```
Add an empty line between.